File: swap.js
function main() {
    let a = 100, b = 200;
    console.log(`After swapping, a = ${a} and b = ${b}`);

    // 1. Swapping using three Variables
    let temp = a;
    a = b;
    b = temp;
    console.log(`After swapping, a = ${a} and b = ${b}`);

    // 2. Using Two Variables
    a = a + b;
    b = a - b;
    a = a - b;
    console.log(`After swapping, a = ${a} and b = ${b}`);

    // 3. Swapping a and b using XOR
    a = a ^ b;
    b = a ^ b;
    a = a ^ b;
    console.log(`After swapping, a = ${a} and b = ${b}`);
}

// Calling the main function
main();

File: swap.java
public class swap {
    public static void main(String[] args) {
        int a = 3;
        int b = 2;
        System.out.println("Before Swap: ");
        System.out.println("a = "+a);
        System.out.println("b = "+b);

        a = a + b;
        b = a - b;
        a = a - b;

        System.out.println("After Swap: ");
        System.out.println("a = "+a);
        System.out.println("b = "+b);
    }
    
}
